Add MS-ISAC Event fetching (#40230)
demisto/content: Packs/MS-ISAC/ReleaseNotes/1_0_5.md

#### Integrations

##### MS-ISAC

> - Added support for **Fetch incidents** parameter.
> - Added support for **First Fetch Time** parameter that when the integration intstance is configured to fetch for the first time, this parameter will determine how far back it will look to fetch.
> - Added support for **Incidents Fetch Interval** parameter.

Reviewer: These are all standard for fetch-incidents, a single line indicating fetch incidents support was added is enough.
```suggestion
- Added support for ***fetch-incidents*** to periodically fetch albert events.
```

Author: Ok, I can change that. I normally just rely on the auto-creation of my release notes, and clean anything up from there.

> - Added support for the API key to be a credential object.

Reviewer: ```suggestion
- Updated the *API key* instance parameter to support the use of credential objects.
```
